### PR TITLE
Next layout

### PR DESCRIPTION
**Pull request by aggre**
## Proposed Changes

- The previous layout could not withstand the increase in pages(routes). So I implemented a new layout.
- Sorry, this PR is so BIG change!

Screenshot_2020-07-14 Stakes social
Screenshot_2020-07-14 Stakes social(1)
Screenshot_2020-07-14 Stakes social(2)

## Implementation

- Many layouts and some components structure

### CODE REVIEW

File: packages/web/src/components/molecules/Navigation/index.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  )
+
+  return (
+    <NavMenu onClick={handleClick} selectedKeys={[current]} mode="horizontal">
+      {navs.map(nav => (
+        <NavMenu.Item key={nav.key}>{nav.label}</NavMenu.Item>
+      ))}
+    </NavMenu>
+  )
+}
```
Comment (abyssparanoia) [inline packages/web/src/components/molecules/Navigation/index.tsx:48]: In Next.js, except for situations where the application actively transitions, you should use basically Link Component. If you don't, no prefetch will be done and the performance of the transition will be extremely slow. 

Therefore, I thought you should use the nav manu item the Link Component of next js.

Comment (aggre) [reply packages/web/src/components/molecules/Navigation/index.tsx:48]: Okay, I tried to use the Link component, but the Menu component was broken🤔

File: packages/web/src/components/molecules/Navigation/index.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    },
+    [setCurrent]
+  )
+
+  return (
+    <NavMenu onClick={handleClick} selectedKeys={[current]} mode="horizontal">
+      {navs.map(nav => (
+        <NavMenu.Item key={nav.key}>
+          <Link href={nav.pathname}>{nav.label}</Link>
+        </NavMenu.Item>
```
Comment (aggre) [inline packages/web/src/components/molecules/Navigation/index.tsx:44]: @abyssparanoia I could fix a Navigation component!

**Comment by aggre** [comment]
I fixed the problem so I merge this, if there is a problem I can revert this :)

**Pull request merged**